privacyidea/api/before_after.py

```diff
@@ -266,7 +266,21 @@ def after_request(response):
 @postrequest(sign_response, request=request)
 def auth_error(error):
     if "audit_object" in g:
-        g.audit_object.log({"info": error.message})
+        message = ''
+        description = ''
+        
+        if hasattr(error, 'message'):
+            message = error.message
+            
+        if hasattr(error, 'description'):
+            message = "{}|{}".format(message, error.description)
+
+        if hasattr(error, 'details'):
+            if error.details:
+                if 'message' in error.details:
+                    message = "{}|{}".format(message, error.details['message'])
```

Comment:
Could you change this to u``"{}|{}"``? I think ``error.details`` is a unicode string and could contain non-ASCII characters, which would break without the ``u`` prefix to the format string literal.

Reply:
https://github.com/privacyidea/privacyidea/pull/1197/files#diff-522d037024042d4c2a1037c637df32d9R262